src/vtlengine/Operators/Analytic.py

```diff
@@ -106,14 +110,24 @@ def validate(  # type: ignore[override]
             measures = operand.get_measures()
             if len(measures) == 0:
                 raise SemanticError("1-1-1-8", op=cls.op, name=operand.name)
+
+            if cls.type_to_check is None and cls.op in [MAX, MIN, SUM]:
+                is_Number = False
+                for measure in measures:
+                    is_Number |= isinstance(measure.data_type, Number)
+                cls.type_to_check = Number if is_Number else Integer
```

Comment:
We cannot change the class attribute, we must use another variable at instance level, or at least add an else clause to ensure we keep it at None or Number